bbn-test-5/babylon-node/README.md

```diff
@@ -111,30 +111,39 @@ Note: If you're running a validator or RPC node that needs to handle queries,
 it's recommended to keep these default values for optimal performance. Only 
 adjust these if you're running a node with limited memory resources.
 
-2. On `config.toml`, populate your seed nodes using entries from the 
-[network page](../README.md#seed-nodes):
+2. On `config.toml`, update the the following settings:
 
 ```shell
-# P2P Configuration Options    
+[p2p]
 
-# This is only an example and should be replaced with actual testnet seed
-# nodes.
-# Comma separated list of seed nodes to connect to
-seeds = "8fa2d1ab10dfd99a51703ba760f0ef555ae88f36@16.162.207.201:26656" 
+# These are placeholder values and should be replaced
+seeds = "NODE_ID1@NODE_ENDPOINT1:PORT1,NODE_ID2@NODE_ENDPOINT2:PORT2"
+
+# These are placeholder values and should be replaced
+persistent_peers = "NODE_ID1@NODE_ENDPOINT1:PORT1,NODE_ID2@NODE_ENDPOINT2:PORT2"
+
+[consensus]
+
+timeout_commit = "10s"
 ```
 
 Parameters:
 - `seeds`: Comma separated list of seed nodes that your node will connect to for 
-discovering other peers in the network
+discovering other peers in the network; you can obtain seed endpoints from
+[here](../README.md#seed-nodes)
+- `persistent_peers`: Comma separated list of nodes that your node will use as
+persistent peers; you can obtain peers from [here](../README.md#peers)
+- `timeout_commit`: The Babylon network block time
```

Comment:
Let's highlight this somehow? That the user must set it

Reply:
sure